api_spec.yaml

```diff
@@ -3246,9 +3246,497 @@ paths:
+
+        If `tenantId` is specified and is not equal to `defaultTenantId`, SuperTokens subscription license key is required.
+
+        If `tenantId` is not provided, `defaultTenantId` will be used.
+
+      operationId: deleteTenantConfigPOST
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - thirdPartyId
+              properties:
+                thirdPartyId:
+                  $ref: '#/components/schemas/thirdPartyId'
+                tenantId:
+                  $ref: '#/components/schemas/tenantId'
+
+      responses:
+        200:
+          description: Remove thirdparty config result
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - type: object
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    didConfigExist:
+                      type: boolean
+        '400':
+          $ref: '#/components/responses/400'
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/list:
+    get:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Returns the list of thirdparty provider config across all tenants for the given `thirdPartyId`
+
+      operationId: listConfigsGET
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+        - name: thirdPartyId
+          in: query
+          required: true
+          schema:
+            $ref: '#/components/schemas/thirdPartyId'
+      responses:
+        200:
+          description: Thirdparty Provider Config list
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - type: object
+                  required:
+                    - status
+                    - providers
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    providers:
+                      type: array
+                      items:
+                        $ref: '#/components/schemas/thirdPartyProviderConfig'
+                - type: object
+                  required:
+                    - status
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusTenantDoesNotExistError'
```

Comment:
- remove oneOf
- This API cannot return `statusTenantDoesNotExistError` cause it doesn't take tenantId as an input

Reply:
Done